R/make.known.table.R

```diff
@@ -1,115 +0,0 @@
-#' @export
-make.known.table <-
-function(metabolite.table, adduct.table, ion.mode="+")
-###		the metabolite.table should be a four-column data frame. The columns should be chemical formula, HMDB_ID, KEGG ID, and monoisotopic mass
-###		The two IDs can be substitute by others
-###		the adduct table should be a four-column data frame. The columns are adduct type, mass divider (absolute value of the number of charges), mass addition (after dividing), and charge of the ion form.
```

Comment:
This functionality is actually also in some way implemented in `recetox-xmsannotator` - is it really not needed in this repository? Was it for sure not somehow used in the hybrid?